index.html

```diff
@@ -14,164 +14,197 @@
-                    <li>&nbsp;&nbsp;2015</li>
+                <h1>Multi-Post<br>Stories</h1>
+                <div>
+                    <h4>CANOPY</h4>
+                    <ul>
+                        <li><img src='./images/Counter.svg' alt="Counter image"></li>
+                        <li>&nbsp;&nbsp;Back End Dev</li>
+                        <li>&nbsp;&nbsp;<img src='./images/Counter.svg' alt="Counter image"></li>
+                        <li>&nbsp;&nbsp;2015</li>
+                    </ul>
+                </div>
+                <p class="p_mobile">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.</p>
+                <p class="p_desktop">A daily selection of privately personalized reads; no accounts <br>or sign-ups required.</p>
+                <ul class="diff_classes">
+                    <li><h4><a href=#>html</a></h4></li>
+                    <li><h4><a href=#>css</a></h4></li>
+                    <li><h4><a href=#>javaScript</a></h4></li>
                 </ul>
+                <a href=#>See Project</a>
             </div>
-            <p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.</p>
-            <ul class="diff_classes">
-                <li><h4><a href=#>html</a></h4></li>
-                <li><h4><a href=#>css</a></h4></li>
-                <li><h4><a href=#>javaScript</a></h4></li>
-            </ul>
-            <a href=#>See Project</a>
         </div>
     </div>
-    <section class="container about_section">
-        <h1>About<br>Myself</h1>
-        <p>Hello I'm a software developer! I can help<br>
-            you build a product, feature or website<br>
-            Look through some of my work and<br>
-            experience! If you like what you see and<br>
-            have a project you need coded, don't<br>
-            hesitate to contact me.</p>
-        <h4>LET'S CONNECT</h4>
-        <ul class="social_media_icons">
-            <li><a href=#><img src='./images/twitter.svg' alt="twitter icon"></a></li>
-            <li><a href=#><img src='./images/Linkedin_icon.svg' alt="LinkedIn icon"></a></li>
-            <li><a href=#><img src='./images/m_icon.svg' alt="M icon"></a></li>
-            <li><a href=#><img src='./images/cat_icon.svg' alt="Cat icon"></a></li>
-            <li><a href=#><img src='./images/hand_icon.svg' alt="Hand icon"></a></li>
-        </ul>
-        <a href=#>Get my Resume</a>
-    </section>
-    <section class="container">
-        <div class="all_fm_skill">
-            <div class="lang_section">
-                <h2>Languages</h2>
-                <i class="fa fa-angle-down"></i>
-            </div>
-            <div class="js_flex1">
-                <div class="js_flex">
-                    <img src='./images/Ellipse_1.svg' alt="js_logo">
-                    <h3>&nbsp;&nbsp;JavaScript</h3>
+    <div class="get_lang_flex">
+        <section class="container about_section">
```

Comment:
- [ ] Please there is a margin between "About myself" section and the projects section. Please kindly fix that. See image below;

***The design***
![medD2](https://user-images.githubusercontent.com/65852621/135705182-ce792e0c-ef02-484e-9a4f-d202b531c4e7.png)

***Your implementation***
![medI](https://user-images.githubusercontent.com/65852621/135705197-da921107-32f9-484a-bcd9-862d4a762793.png)

Reply:
I have corrected this design issue in my code. Thank you.

Reply:
I have also deleted unnecessary files from my repository as per the reviewer's suggestion. Thank you.